clients/v1_auth/v1_auth.go

```diff
@@ -0,0 +1,423 @@
+		token: &v1Token{
+			ID:          auth.GetId(),
+			Description: auth.GetDescription(),
+			Token:       auth.GetToken(),
+			Status:      auth.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: ps,
+		},
+	})
+}
+
+type SetPasswordParams struct {
+	clients.AuthLookupParams
+	Password string
+}
+
+func (c Client) SetPassword(ctx context.Context, params *SetPasswordParams) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	auth, err := c.LegacyAuthorizationsApi.GetAuthorizationsID(ctx, id).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find authorization with User ID %q: %w", id, err)
+	}
+
+	password := params.Password
+	if password == "" {
+		pass, err := c.StdIO.GetPassword("Please type new password")
+		if err != nil {
+			return err
+		}
+		password = pass
+	}
+
+	err = c.LegacyAuthorizationsApi.SetAuthorizationsIDPassword(ctx, auth.GetId()).Password(password).Execute()
+	if err != nil {
+		return fmt.Errorf("error setting password: %w", err)
+	}
+
+	return nil
+}
+
+func (c Client) printV1Tokens(params *v1PrintOpts) error {
+	if c.PrintAsJSON {
+		var v interface{}
+		if params.token != nil {
+			v = params.token
+		} else {
+			v = params.tokens
+		}
+		return c.PrintJSON(v)
+	}
+
+	headers := []string{
+		"ID",
+		"Description",
+		"Name / Token",
+		"User Name",
+		"User ID",
+		"Permissions",
+	}
+	if params.deleted {
+		headers = append(headers, "Deleted")
+	}
+	if params.token != nil {
+		params.tokens = append(params.tokens, *params.token)
+	}
+
+	var rows []map[string]interface{}
+	for _, u := range params.tokens {
+		row := map[string]interface{}{
+			"ID":           u.ID,
+			"Description":  u.Description,
+			"Name / Token": u.Token,
+			"User Name":    u.UserName,
+			"User ID":      u.UserID,
+			"Permissions":  u.Permissions,
+		}
+		if params.deleted {
+			row["Deleted"] = true
+		}
+		rows = append(rows, row)
+	}
+	return c.PrintTable(headers, rows...)
+}
+
+func permString(p api.Permission) string {
+	ret := p.GetAction() + ":"
+	r := p.GetResource()
+
+	if r.GetOrgID() != "" {
+		ret += "orgs/" + r.GetOrgID()
+	}
+	ret += "/" + r.GetType()
+	if r.GetId() != "" {
+		ret += "/" + r.GetId()
+	}
+	return ret
+}
+
+func (c Client) getAuthReqID(ctx context.Context, params *clients.AuthLookupParams) (id string, err error) {
+	if params.ID.Valid() {
+		id = params.ID.String()
+	} else {
+		auths, err := c.LegacyAuthorizationsApi.GetAuthorizations(ctx).Token(params.Username).Execute()
+		if err != nil || len(auths.GetAuthorizations()) == 0 {
+			err = fmt.Errorf("could not find v1 auth with token (username) %q: %w", params.Username, err)
+		} else {
+			id = auths.GetAuthorizations()[0].GetId()
+		}
+	}
+	return
+}
+
+func (c Client) getOrgID(ctx context.Context, params clients.OrgParams) (string, error) {
+	if params.OrgID.Valid() || params.OrgName != "" || c.ActiveConfig.Org != "" {
```

Comment:
```suggestion
	if !params.OrgID.Valid() && params.OrgName == "" && c.ActiveConfig.Org == "" {
```
The current logic causes the a failure if org is specified through any one of the 3